standalone/pruntime/enclave/Cargo.toml

```diff
@@ -41,6 +51,13 @@ sgx_tstd        = { rev = "v1.1.3", git = "https://github.com/apache/teaclave-sg
 [patch.crates-io]
 ppv-lite86 = { path = "../../../cryptocorrosion-sgx/utils-simd/ppv-lite86/" }
 ring = { git = "https://github.com/Phala-Network/ring-sgx.git", branch = "phala" }
+async-io = { git = "https://github.com/Phala-Network/async-io-sgx.git", branch = "phala" }
+parking = { git = "https://github.com/Phala-Network/parking-sgx.git", branch = "phala" }
+httparse = { git = "https://github.com/Phala-Network/httparse-sgx.git", branch = "phala" }
+aho-corasick = { git = "https://github.com/Phala-Network/aho-corasick-sgx.git", branch = "phala" }
+futures-executor = { git = "https://github.com/Phala-Network/futures-rs-sgx.git", branch = "phala" }
+event-listener = { git = "https://github.com/Phala-Network/event-listener-sgx.git", branch = "phala" }
+parking_lot_core = { git = "https://github.com/Phala-Network/parking_lot-sgx.git", branch = "phala" }
```

Comment:
Do we patch them mostly because of some weird syscall? I see ac-automata here, but it sounds like just a pure algorithm.

Reply:
Patching httparse & aho-corasick for CPUID usage.
Others for SYS_futex syscall usage. We can not find a way to implement the futex in enclave so far.

Reply:
Got it. No problem